tests/modules/programs/firefox/floorp-with-firefox.nix

```
@@ -0,0 +1,16 @@
+{ config, lib, realPkgs, ... }:
+
+lib.mkIf config.test.enableBig {
+  programs.firefox = {
+    enable = true;
+    package = realPkgs.firefox-unwrapped;
```

```suggestion
    package = null;
```
We don't actually need to include the packages to test the evaluation for this scenario, I believe.